e2e-tests/data-at-rest-encryption/run

```diff
@@ -57,10 +57,14 @@ sleep 5
 
 desc "check backup and restore -- minio"
 backup_dest_minio=$(get_backup_dest "$backup_name_minio")
-kubectl_bin run -i --rm aws-cli --image=perconalab/awscli --restart=Never -- \
-	/usr/bin/env AWS_ACCESS_KEY_ID=some-access-key AWS_SECRET_ACCESS_KEY=some-secret-key AWS_DEFAULT_REGION=us-east-1 \
-	/usr/bin/aws --endpoint-url http://minio-service:9000 s3 ls s3://${backup_dest_minio}/rs0/ \
+retry 3 8 kubectl_bin run -i --rm aws-cli --image=amazon/aws-cli \
+	--restart=Never \
+	--env=AWS_ACCESS_KEY_ID=some-access-key \
+	--env=AWS_SECRET_ACCESS_KEY=some-secret-key \
+	--env=AWS_DEFAULT_REGION=us-east-1 -- \
+	--endpoint-url http://minio-service:9000 s3 ls s3://${backup_dest_minio}/rs0/ \
```

Comment:
if we retry here, do we need to retry in other places? and maybe we can move this to `e2e-test/functions` as I see we use same flags everywhere